src/ui.js

```diff
@@ -318,14 +351,18 @@ function init() {
 
     populateRatesDropDown();
 
+    restartBtn.disabled = true;
+    restartBtn.addEventListener("click", restart);
+
     pauseBtn.addEventListener("click", togglePause);
     ratesDropDown.addEventListener("change", () => ratesDropDownOnChange(ratesDropDown.value));
-
     dumpStateBtn.addEventListener("click", () => {
         simulation.patchAugmintParams(getParamsFromUI());
         logger.print(simulation.getState());
     });
-
+    dumpIterationLogBtn.addEventListener("click", () => {
```

Comment:
same here: I think I've removed iteration log dump. maybe it sneaked back in a merge you did?

Reply:
removed